NuGet.config

```diff
@@ -18,5 +19,12 @@
     <add key="myget-legacy" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/myget-legacy/nuget/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
   </packageSources>
-  <disabledPackageSources />
+  <disabledPackageSources>
+    <add key="darc-int-dotnet-extensions-ce485be" value="true" />
+    <add key="darc-int-dotnet-extensions-8be2e8d" value="true" />
+    <add key="darc-int-dotnet-extensions-77d62c5" value="true" />
```

Comment:
nit: Why are more feeds disabled for extensions than listed above❔

Reply:
That's a darc quirk, I'm not sure what causes it - @mmitche?